practices/Super-Gabriel/practica03.py

```diff
@@ -0,0 +1,113 @@
+import nltk
+from nltk.corpus import stopwords
+
+""" ------------------------ Parte 1 --------------------------- """
+
+#descargando los stopwords de nltk
+nltk.download('stopwords')
+
+# guradando solo los primeros 100 elementos
+nltk_stopwords = stopwords.words('spanish')[:100]
+
+#importando el corpus CREA
+import csv
+
+# Lista para almacenar los datos del CSV
+crea_freq_words = []
+iterador = 0;
+
+with open('../../notebooks/corpora/zipf/crea_full.csv', newline='', encoding='latin-1') as archivo_csv:
+    lector_csv = csv.reader(archivo_csv)
+    
+    for fila in lector_csv:
+        # Agregando la fila como una lista a los datos del CSV
+        crea_freq_words.append(fila[0].split('\t')[0])
+
+        # solo los primeros 100 elementos
+        iterador += 1
+        if iterador >= 100:
+            break
+
+# eliminando espacios de las palabras
+crea_freq_words = [palabra.replace(" ", "") for palabra in crea_freq_words]
+
+# contando coincidencias en ambas listas
+contador = 0
+for word in crea_freq_words[1:]:
+    if word in nltk_stopwords:
+        contador += 1
+
+
+#imprimiendo número de coincidencias
+print(str(contador)+"/100 palabras coinciden del corpus de CREA con los stopwords de nltk")
```

Comment:
Puedes usar f-strings para este tipo de casos:

```suggestion
print(f"{contador}/100 palabras coinciden del corpus de CREA con los stopwords de nltk")
```